src/cli/init.go

```diff
@@ -24,22 +22,9 @@ func initCommand(root *rootCommand) *cobra.Command {
 		Short: shortDescription,
 		Long:  longDescription,
 		PreRunE: func(cmd *cobra.Command, args []string) error {
-			// Ask for the host/token, if they were not specified, to make the first step easier
-			if len(root.options.ApiHost) == 0 {
-				root.options.ApiHost, _ = root.prompt.Ask(&Question{
-					Label:       "API host",
-					Description: "Please enter Keboola Storage API host, eg. \"keboola.connection.com\".",
-					Validator:   apiHostValidator,
-				})
-			}
-			if len(root.options.ApiToken) == 0 {
-				root.options.ApiToken, _ = root.prompt.Ask(&Question{
-					Label:       "API token",
-					Description: "Please enter Keboola Storage API token. The value will be hidden.",
-					Hidden:      true,
-					Validator:   valueRequired,
-				})
-			}
+			// Ask for the host/token, if not specified -> to make the first step easier
+			root.options.AskUser(root.prompt, "ApiHost")
```

Comment:
Presunul som to do metody, kedze mi doslo, ze sa to pouzije aj pri `pull` akcii ... ak si niekto naklonuje uz existujuci repozitar, ale nebude mat nastavene `.env`.